test-http/src/test/cve_tests/cve.py

```diff
@@ -199,6 +201,28 @@ def test_update_nonexistent_cve():
         response_contains_json(res, 'error', 'CVE_RECORD_DNE')
 
 
+def test_record_submission_too_large():
+    """ the cve record cannot be submitted because it is 16 mb"""
+    with open('./src/test/cve_tests/cve_record_fixtures/CVE-2021-0004_largeInput.json') as json_file:
+        data = json.load(json_file)
+        res = requests.post(
+            f'{env.AWG_BASE_URL}{CVE_URL}/CVE-2021-0004',
+            headers=utils.BASE_HEADERS,
+            json=data
+        )
+        assert res.status_code == 413 # payload is too large
```

Comment:
Check the error contents to make sure the message appears as we want it to clients